.github/workflows/comma.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+
+"""
+This is an pretiffy script.
+It runs on lib and test directory to find files
+which doesn't satisfy trailing comma.
+"""
+
+import os
+import sys
+import argparse
+
+
+def arg_parser_resolver():
+    """Resolve, for arguments provided by user."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--dir', type=str, required=False, default=os.getcwd(),
+                        help='directory-location where files are present')
+    return parser.parse_args()
+
+
+def main():
+    """Find, and print, for files having comma in sequence."""
+    args = arg_parser_resolver()
+    # parses through files and saves to a dict
+    file_names_with_size = {}
+
+    # libPath and testPath dir location
+    lib_path = os.path.expanduser(os.path.join(args.dir, 'lib'))
+    test_path = os.path.expanduser(os.path.join(args.dir, 'test'))
+
+    # counting lines in lib dir
+    for root, _, files in os.walk(lib_path, topdown=False):
```

Comment:
You can do the lib dir and test dir together by nesting in another loop

Reply:
ok